src/dispatch.js

```diff
@@ -11,6 +11,42 @@ import {
   ifElse,
 } from 'ramda';
 
+/**
+ * Can be used as a way to compose multiple invokers together to form polymorphic functions,
+ * or functions that exhibit different behaviors based on their argument(s).
+ * Consumes dispatching functions and keep trying to invoke each in turn, until a non-nil value is returned.
```

Comment:
Consumes a list of dispatching functions and tries each in turn until a non-nil value is returned.